invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/deposit.less

```diff
@@ -129,3 +129,74 @@
 .item div.description:not(.license-item-description) {
   display: initial !important;
 }
+
+/* Styling for the community selection modal */
+
+.community-selection-modal {
+  height: 95vh;
+  flex-direction: column;
+  justify-content: space-between;
+  margin: 0 !important;
```

Comment:
this and all the other `!important` usages need to be evaluated and removed if possible